lib/browser/api/web-contents.ts

```diff
@@ -537,6 +537,14 @@ const addReturnValueToEvent = (event: Electron.IpcMainEvent) => {
   });
 };
 
+const getWebFrameForEvent = (event: Electron.IpcMainEvent | Electron.IpcMainInvokeEvent) => {
+  try {
+    return webFrameMainBinding.fromIdOrNull(event.processId, event.frameId);
+  } catch {
+    return null;
+  }
```

Comment:
We don't need a `try` / `catch`.  Just to check if `event.processId` and `event.frameId` are defined